modules/core/test/v2/unit/coins/cspr.ts

```diff
@@ -1,20 +1,24 @@
-import * as Promise from 'bluebird';
-const co = Promise.coroutine;
-
+import { Promise as BluebirdPromise } from 'bluebird'; 
+import { Cspr as CsprAccountLib, register } from '@bitgo/account-lib';
 import { TestBitGo } from '../../../lib/test_bitgo';
 import { Tcspr } from '../../../../src/v2/coins';
 import { Cspr } from '../../../../src/v2/coins';
+import { Transaction } from '@bitgo/account-lib/dist/src/coin/cspr/transaction';
```

Comment:
How come this needs a path import? In general, this is something that should be avoided - if the type isn't exported from the package, it probably shouldn't be used by consumers of the package.

Since it's a test I won't block the PR, but please keep this in mind in the future.